### ISSUE TITLE
NVDA switches into scan mode after navigating to a separator

### ISSUE THREAD

**Issue by jdrush89** [nvaccess/nvda#11403]
### Steps to reproduce:
The issue can be observer here https://codesandbox.io/s/suspicious-morse-bf957?file=/example.tsx
1. Make sure NVDA is not in scan mode
2. Load codesandbox example
3. Tab to the separator in the Splitter
4. Tab to the list on the right side of the splitter
5. Navigate up and down with the arrow keys
### Actual behavior:
NVDA has put itself into scan mode, using the arrow keys navigates through each cell in the list
### Expected behavior:
NVDA should not be in scan mode, using the arrow keys should navigate through each row.
#### NVDA installed/portable/running from source:
installed
#### NVDA version:
2019.3
#### Windows version:
windows 10
#### Does the issue still occur after restarting your computer?
yes

**Comment by Adriani90** [comment]
I saw this issue in many examples, this happens only when you let NVDA switch automatically to focus mode when the focus changes. If you switch manually to focus mode with nvda+space bar, then NVDA will not switch back to browse mode when you tab into the region at the separator and you will be able to access the list.

@feerrenrut, @jcsteh, This is reproducible in every browser.

Starting at the edit field "address bar input", NVDA is in focus mode. Pressing tab will switch to browse mode on the unlabeled buttons. Pressing tab again will swich to focus mode on the tab area, and pressing tab a last time will switch back to browse mode on the separator region. In that region is a list, but NVDA does not identify it in the virtual buffer, so I guess this is why it does not switch to focus mode when tabbing into it.

I am not sure if this list is exposed correctly by the browser or if NVDA fails here.

**Comment by akshansh04** [comment]
@Adriani90 - Can you confirm if this bug is related to NVDA? We have a bug related to this and are looking at NVDA acceptance on the same.

**Comment by Adriani90** [comment]
@jdrush89, @akshansh04 the minimal test case in the description seems not to work anymore, please provide an updated minimal test case.
Do you still have this issue in NVDA 2024.2 Stable or NVDA 2024.3 Beta?

**Comment by Adriani90** [comment]
We didn't get any updates from people who could reproduce this since more than 4 years, closing as abandoned. Please comment with an updated minimal test case and we can reopen, in case the issue still exists.
